src/current/releases/release-support-policy.md

```diff
@@ -119,8 +142,12 @@ Innovation releases are not eligible for Assistance Support, and reach EOL at th
   </tr>
       {% endif %}
 
-  <tr>{% comment %} Always print a GA row. For 23.2+, add a link to the first footnote{% endcomment %}
-    <td><a href="{% link releases/{{ v.major_version }}.md %}">{{ v.major_version }}{% if will_never_have_lts == false and v.initial_lts_patch == "N/A" %}&nbsp;<a href="#lts-tbd"><sup>*</sup></a>{% endif %}</td>
+  {% comment %} Always print a GA row.
+    For regular releases not yet in LTS, add a link to the first footnote
+    For currently supported skippable releases, add a link to the second footnote
+  {% endcomment %}
+  <tr>
+    <td><a href="{% link releases/{{ v.major_version }}.md %}">{{ v.major_version }}{% if will_never_have_lts == false and v.initial_lts_patch == "N/A" %}&nbsp;<a href="#lts-tbd"><sup>*</sup></a>{% elsif skippable == true %}&nbsp;<a href="#skippable"><sup>**</sup></a>{% endif %}</td>
```

Comment:
@mdlinville Not seeing a 24.2 row in the [preview](https://deploy-preview-18778--cockroachdb-docs.netlify.app/docs/releases/release-support-policy.html#supported-versions)

Reply:
DOne